net/acme-common/files/acme.sh

```diff
@@ -107,11 +104,19 @@ load_globals() {
 		log err "account_email option is required"
 		exit 1
 	fi
+	export account_email
+
+	config_get state_dir "$section" state_dir
+	if [ "$state_dir" ]; then
```

Comment:
I thought about this one, and found warning on non-empty value to be better, since that means the user has the option set in the config, and could give an impression that it could be changed to other values.